From 799d4421d99c4d58bdc5d58bd777fe68bda4f498 Mon Sep 17 00:00:00 2001
From: Biwen Li <biwen.li@nxp.com>
Date: Fri, 9 Aug 2019 15:49:29 +0800
Subject: [PATCH 576/741] Revert "soc: fsl: add support for arm64/Layerscape
 SOC specific drivers"

commit 7316dd1da1a173b60ea95c1c86ab995d1830d123 from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

This reverts commit 7900931b034d93ea3ba1e241b0012853d4a5b502.

Fix conflict files:
	driver/soc/fsl/Kconfig
	driver/soc/fsl/Makefile

Dest:
	drop old flextimer driver to fix compilation error.
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/soc/fsl/Kconfig             |  4 ----
 drivers/soc/fsl/Makefile            |  1 -
 drivers/soc/fsl/layerscape/Kconfig  | 10 ----------
 drivers/soc/fsl/layerscape/Makefile |  1 -
 4 files changed, 16 deletions(-)
 delete mode 100644 drivers/soc/fsl/layerscape/Kconfig
 delete mode 100644 drivers/soc/fsl/layerscape/Makefile

diff --git a/drivers/soc/fsl/Kconfig b/drivers/soc/fsl/Kconfig
index 2ea999f8c480..86974116dd54 100644
--- a/drivers/soc/fsl/Kconfig
+++ b/drivers/soc/fsl/Kconfig
@@ -48,8 +48,4 @@ config FSL_SLEEP_FSM
 	  This driver configures a hardware FSM (Finite State Machine) for deep sleep.
 	  The FSM is used to finish clean-ups at the last stage of system entering deep
 	  sleep, and also wakes up system when a wake up event happens.
-
-if ARM || ARM64
-source "drivers/soc/fsl/Kconfig.arm"
-endif
 endmenu
diff --git a/drivers/soc/fsl/Makefile b/drivers/soc/fsl/Makefile
index b3f3072e52bf..af70a300bc02 100644
--- a/drivers/soc/fsl/Makefile
+++ b/drivers/soc/fsl/Makefile
@@ -10,5 +10,4 @@ obj-$(CONFIG_FSL_QIXIS) 		+= qixis_ctrl.o
 obj-$(CONFIG_FSL_GUTS)			+= guts.o
 obj-$(CONFIG_FSL_MC_DPIO) 		+= dpio/
 obj-$(CONFIG_FSL_LS2_CONSOLE)		+= ls2-console/
-obj-$(CONFIG_LS_SOC_DRIVERS)		+= layerscape/
 obj-$(CONFIG_FSL_SLEEP_FSM)	+= sleep_fsm.o
diff --git a/drivers/soc/fsl/layerscape/Kconfig b/drivers/soc/fsl/layerscape/Kconfig
deleted file mode 100644
index e1373aa1b4ea..000000000000
--- a/drivers/soc/fsl/layerscape/Kconfig
+++ /dev/null
@@ -1,10 +0,0 @@
-#
-# Layerscape Soc drivers
-#
-config  FTM_ALARM
-	bool "FTM alarm driver"
-	default n
-	help
-	Say y here to enable FTM alarm support.  The FTM alarm provides
-	alarm functions for wakeup system from deep sleep.  There is only
-	one FTM can be used in ALARM(FTM 0).
diff --git a/drivers/soc/fsl/layerscape/Makefile b/drivers/soc/fsl/layerscape/Makefile
deleted file mode 100644
index 6299aa1de215..000000000000
--- a/drivers/soc/fsl/layerscape/Makefile
+++ /dev/null
@@ -1 +0,0 @@
-obj-$(CONFIG_FTM_ALARM) += ftm_alarm.o
-- 
2.17.1

